daemon/config/config_test.go

```diff
@@ -48,7 +59,168 @@ func TestFlattenConfig(t *testing.T) {
 }
 
 func TestConfigValidate(t *testing.T) {
-	// TODO
+	assert := assert.New(t)
+	// Test default configuration
+	cfg := &Config{}
+	assert.Equal(nil, cfg.Validate())
+
+	// Test volume configuration
+	cfg = &Config{
+		VolumeConfig: volume.Config{
+			RemoveVolume:   false,
+			DefaultBackend: "/path/to",
+			VolumeMetaPath: "/foo/bar",
+			DriverAlias:    "local=alilocal",
+		},
+	}
+	assert.Equal(nil, cfg.Validate())
+
+	// Test network configuration
+	cfg = &Config{
```

Comment:
@chuanchang The config validate only checks the labels and duplicate listeners. If we don't add more checker, the test case will be no-op here. Could you help to do that?

https://github.com/alibaba/pouch/blob/master/daemon/config/config.go#L130-L162

Reply:
sure, I know this part code, but I lost this scenario when wrote testing, the next version will cover it soon.